lib/selection-count-view.coffee

```diff
@@ -1,15 +1,28 @@
+_ = require 'underscore-plus'
+
 class SelectionCountView extends HTMLElement
+  toolTipDisposable: null
```

Comment:
Actually, sorry, after thinking about this more I don't think it should be declared at like this unless the other disposables are as well like `configSubscription`.

So for consistency, I'd say either all or none, and for initial simplicity I'd say just none of them.